app/policies/application_policy.rb

```diff
@@ -1,8 +1,35 @@
-class ApplicationPolicy
-  attr_reader :user, :record
+class ApplicationPolicy < ActionPolicy::Base
+  authorize :user, allow_nil: true
 
-  def initialize(user, record)
-    @user = user
-    @record = record.to_model
+  def index?
```

Comment:
What do you think about using aliases here from the Action Policy documentation.
```ruby
alias_rule :new?, to: :create?
alias_rule :edit?, to: :update?
```